master/internal/job/job.go

```diff
@@ -216,8 +218,14 @@ func (j *Jobs) getJobs(ctx *actor.Context, resourcePool string, desc bool) ([]*j
 	for jID, jRMInfo := range jobQ {
 		v1Job, ok := jobs[jID]
 		if ok {
+			// interesting that the update is a side effect
+			// of the getJobs function. I am guessing that
+			// I should leave it, regardless of filters?
 			UpdateJobQInfo(v1Job, jRMInfo)
```

Comment:
I believe this is to make sure the job queue info is as up to date as possible

Reply:
https://user-images.githubusercontent.com/28383080/184258065-5bb6fdbb-a056-42ec-ac99-48bf2a3cfe17.mp4

this was the issue this PR is intended to resolve

Reply:
that does look off though, i agree. is that a regression?

Reply:
the pagination issue shown in the video. I see that in master as well